pkg/datapath/linux/node.go

```diff
@@ -695,7 +695,7 @@ func (n *linuxNodeHandler) insertNeighbor(ctx context.Context, newNode *nodeType
 						logfields.IPAddr:       neigh.IP,
 						logfields.HardwareAddr: neigh.HardwareAddr,
 						logfields.LinkIndex:    neigh.LinkIndex,
-					}).WithError(err).Warn("Failed to remove neighbor entry")
+					}).WithError(err).Info("Failed to remove neighbor entry")
```

Comment:
why would this happen and not be an error? Is it possible for the entry to be removed async to this operation so it does not exist or something?

Reply:
I was thinking about the case when a network operator removes an entry manually.